schema.yml

```diff
@@ -0,0 +1,831 @@
+      tags:
+        - datasets
+    post:
+      summary: Alter a dataset's metadata
+      description: |
+        Alter a dataset's metadata
+      operationId: dataset_operations.post_metadata
+      parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          title: Dataset Id
+          type: string
+      responses:
+        200:
+          description: Posted metadata
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatasetMetadataBody'
+        400:
+          $ref: "#/components/responses/400BadRequestError"
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        403:
+          $ref: "#/components/responses/403ForbiddenError"
+        404:
+          $ref: "#/components/responses/404NotFoundError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - datasets
+  /datasets/statistics:
+    get:
+      summary: Grab summary statistics for the datasets.
+      description: |
+        Gets an object of statistics for all datasets.
+      operationId: dataset_operations.statistics
+      responses:
+        200:
+          description: Retrieved dataset information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatasetStatsBody'
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - datasets
+  /datasets/{id}/statistics:
+    get:
+      summary: Grab summary statistics for the given dataset
+      description: |
+        Gets an object of statistics for the given dataset
+      operationId: dataset_operations.statistics_by_id
+      parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          title: Dataset Id
+          type: string
+      responses:
+        200:
+          description: Retrieved dataset information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatasetStatsBody'
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - datasets
+
+  /persons:
+    get:
+      summary: List persons. With parameters, searches for all persons that match the input.
+      description: |
+        Gets a list of all persons. When a search parameter is given, this will search through all
+        persons instead. The granularity of the response is dependent on the authorized level
+        of access of the user, but usually returns only summary-level information. To obtain more
+        information about a person, use the /persons/{id} endpoint.
+      operationId: person_operations.search
+      parameters:
```

Comment:
same as datasets, we may not need search here if we can do this all via beacon. But if we do, we don't want to hardcode all of the search parameters